src/devices/acurite.c

```diff
@@ -745,6 +748,40 @@ static int acurite_txr_decode(r_device *decoder, bitbuffer_t *bitbuffer)
             valid++;
         }
 
+        // 515 sensor messages are 6 bytes.
+        if (browlen == ACURITE_515_BITLEN / 8) {
+        // Sensor type is determined by bit 0 from the message_type.
+        // 0 = refrigerator, 1 = freezer
+        sensor_type = message_type & 1 ? 'F' : 'R';
```

Comment:
If it appears that message_types 8 and 9 have essentially been assigned to this senor, then I recommend testing all 6 bits of the message type to avoid false positives.

Add #defines with the other message types for 8 and 9.

Reply:
Fixed.